core/src/test/scala/unit/kafka/cluster/AssignmentStateTest.scala

```diff
@@ -29,53 +29,53 @@ object AssignmentStateTest extends AbstractPartitionTest {
     Arguments.of(
       List[Integer](brokerId, brokerId + 1, brokerId + 2),
       List[Integer](brokerId, brokerId + 1, brokerId + 2),
-      List.empty[Integer], List.empty[Integer], Seq.empty[Int], false),
+      List.empty[Integer], List.empty[Integer], Seq.empty[Int], Boolean.box(false)),
```

Comment:
For the future, I think it's a bit cleaner to do `false: java.lang.Boolean`. The compiler figures out the most appropriate way to convert.

Reply:
I’ll keep it in mind.